### DISCUSSION TITLE
How do you disable notifications?

### DISCUSSION THREAD

**Discussion by bugz8unny69** [abraunegg/onedrive#2217] [Q&A] [answered]
With the recent change to onedrive client with #2153, I'm getting a notification every 45 seconds due to invalid Microsoft Naming Convention errors. I've gone through the documentation and maybe I've missed it but how do you disable notifications?

**Comment by abraunegg** [comment, marked as answer]
@lhorace 
> I'm getting a notification every 45 seconds due to invalid Microsoft Naming Convention errors.

Every 45 seconds is a *bit* much ..... why are you syncing every 45 seconds when the default is 300 seconds - please can you check your configuration with `--display-config`

You *really* should fix your files that are 'invalid' due to naming conventions ....

> I've gone through the documentation and maybe I've missed it but how do you disable notifications?

I guess you missed this: https://github.com/abraunegg/onedrive/blob/master/docs/USAGE.md#all-available-commands

![image](https://user-images.githubusercontent.com/4956234/201179244-27427fdf-dbc4-4e18-a802-c1ed264bb662.png)

**Reply by bugz8unny69** [reply]
> @lhorace
> 
> > I'm getting a notification every 45 seconds due to invalid Microsoft Naming Convention errors.
> 
> Every 45 seconds is a _bit_ much ..... why are you syncing every 45 seconds when the default is 300 seconds - please can you check your configuration with `--display-config`

Honestly, either I've set it to 45 seconds to reduce data loss or I pull that from example documentation. It's been years since I've touched the configuration.

Configuration file successfully loaded
onedrive version                             = v2.4.21
Config path                                  = /home/suse/.config/onedrive
Config file found in config path             = true
Config option 'sync_dir'                     = /home/suse/Cloud/OneDrive
Config option 'enable_logging'               = false
Config option 'log_dir'                      = /var/log/onedrive/
Config option 'disable_notifications'        = false
Config option 'min_notify_changes'           = 5
Config option 'skip_dir'                     = *.metadata*|LHProjects/Secrets/GNUPG/.gnupg
Config option 'skip_dir_strict_match'        = false
Config option 'skip_file'                    = ./LHProjects/Secrets/ssh/agent-linux*
Config option 'skip_dotfiles'                = false
Config option 'skip_symlinks'                = false
Config option 'monitor_interval'             = 45
Config option 'monitor_log_frequency'        = 5
Config option 'monitor_fullscan_frequency'   = 12
Config option 'read_only_auth_scope'         = false
Config option 'dry_run'                      = false
Config option 'upload_only'                  = false
Config option 'download_only'                = false
Config option 'local_first'                  = false
Config option 'check_nosync'                 = false
Config option 'check_nomount'                = false
Config option 'resync'                       = false
Config option 'resync_auth'                  = false
Config option 'cleanup_local_files'          = false
Config option 'classify_as_big_delete'       = 1000
Config option 'disable_upload_validation'    = false
Config option 'bypass_data_preservation'     = false
Config option 'no_remote_delete'             = false
Config option 'remove_source_files'          = false
Config option 'sync_dir_permissions'         = 700
Config option 'sync_file_permissions'        = 600
Config option 'space_reservation'            = 52428800
Config option 'application_id'               = 
Config option 'azure_ad_endpoint'            = 
Config option 'azure_tenant_id'              = common
Config option 'user_agent'                   = 
Config option 'force_http_11'                = false
Config option 'debug_https'                  = false
Config option 'rate_limit'                   = 0
Config option 'operation_timeout'            = 3600
Config option 'sync_root_files'              = false
Selective sync 'sync_list' configured        = true
sync_list contents:
LHProjects/
Config option 'sync_business_shared_folders' = false
Business Shared Folders configured           = false
Config option 'webhook_enabled'              = false

> You _really_ should fix your files that are 'invalid' due to naming conventions ....

Some of my $HOME folder contents are a symlink to my onedrive. My rudimentary crude backup system.

>  I guess you missed this: https://github.com/abraunegg/onedrive/blob/master/docs/USAGE.md#all-available-commands

Huh, I must have have narrow vision, thanks for pointing it out.

**Reply by abraunegg** [reply]
@lhorace 
> Honestly, either I've set it to 45 seconds to reduce data loss or I pull that from example documentation. It's been years since I've touched the configuration.

A value of 45 was the original 'skilion' value, adopted here 4 years ago but very quickly increased to 300 seconds as a default.

I strongly recommend that you revisit that. Please read:
* https://github.com/abraunegg/onedrive/blob/master/docs/USAGE.md#monitor_interval
* https://github.com/abraunegg/onedrive/blob/master/docs/USAGE.md#monitor_fullscan_frequency

Adjust these values based on your actual data rate of change.

**Reply by bugz8unny69** [reply]
I felt 300 seconds was a bit too long, so I set it to 180 seconds.

**Reply by abraunegg** [reply]
OK .. but is your data locally or online changing every 180 seconds?

**Reply by bugz8unny69** [reply]
Approximately yeah, I make a lot of changes to contents in my onedrive whenever I am on Linux. I use Linux as my workstation.
